k8s/helmfile/helmfile.yaml

```diff
@@ -104,6 +104,16 @@ releases:
   ################################
   # ALL ENVIRONMENTS
   ################################
+  - name: argocd-config
+    namespace: argocd
+    chart: ../../charts/argocd-config
+    values:
+      - env/{{`{{ .Environment.Name }}`}}/base.yaml
+      - env/{{`{{ .Environment.Name }}`}}/private.yaml
```

Comment:
I found out that private.yaml can actually be merged into base.yaml, but since that refactor touches a lot of files I refrained from doing it here